Dockerfile

```diff
@@ -1,6 +1,8 @@
 ARG PYTHON_VERSION=3.11.7-slim
+ARG NVM_VERSION=0.40.0
 
 FROM  --platform=$BUILDPLATFORM python:${PYTHON_VERSION} AS project-base
+ARG NVM_VERSION
```

Comment:
```suggestion
```

Reply:
Is this not needed as it is already declared at line 2?

Reply:
No, you need it else it is not exposed in the FROM section